src/components/FlawTrackers/trackersColumns.ts

```diff
@@ -0,0 +1,66 @@
+export interface TrackerColumn {
+  hasFilter: boolean;
+  index: number;
```

Comment:
Arrays in js mantains the order, there's no need for this index, also seems to be unused